ui/app/components/app/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js

```diff
@@ -182,19 +181,19 @@ export default class AdvancedGasInputs extends Component {
       <div className="advanced-gas-inputs__gas-edit-rows">
         { this.renderGasInput({
           label: this.context.t('gasPrice'),
+          title: this.context.t('gasPriceInfoTooltipContent'),
```

Comment:
We should rename the `title` parameter to `tooltipTitle` for great clarity!

Reply:
This comment should've been made on the `renderGasInput` function declaration but I'm sure you get the idea 😅

Reply:
You got it

Reply:
fixed